nessie/sanity_api_test.go

```diff
@@ -53,8 +53,8 @@ func TestSanityAPI(t *testing.T) {
 	}), nil)
 	require.NoError(t, err, "initial commit")
 
-	log.Debug("list committed files on master")
-	entries = listRepositoryObjects(ctx, t, repo, masterBranch+":HEAD")
```

Comment:
Did we lose functionality or just coverage?

Reply:
un-documented feature - we can add support if needed